service/src/test/java/edu/tamu/catalog/controller/PatronControllerTest.java

```diff
@@ -234,7 +283,7 @@ private void getEndpointNotFound(String sourceUrl, String catalogEndpoint) throw
         restServer.verify();
     }
 
-    private void getEndpointDateParseError(String sourceUrl, String catalogEndpoint) throws Exception  {
+    private void getEndpointDateParseError(String sourceUrl, String catalogEndpoint, Object... pathParameters) throws Exception  {
```

Comment:
Is the pathParameters parameter doing anything here?

Reply:
Nope. Its now removed.